.editorconfig

```diff
@@ -0,0 +1,4 @@
+[*]
+indent_style = space
+indent_size = 4
+end_of_line = crlf
```

Comment:
Yes, I saw the page and understood what it is doing. Was just wondering what editor you were using, since we never had any developer have an issue with this before.